enhancements/single-node/single-node-openshift-day2-workers.md

```diff
@@ -0,0 +1,290 @@
+---
+title: single-node-openshift-day2-workers
+authors:
+  - "@omertuc"
+reviewers:
+  - "@romfreiman"
+  - "@eranco74"
+  - "@tsorya"
+  - TBD
+approvers:
+  - TBD
```

Comment:
It would be good to have someone from the cluster infra team included as reviewer at least. @JoelSpeed can help identify someone.

Which other teams are affected by this work? @sinnykumari do you anticipate any MCO impact?

Reply:
@dhellmann of course. We wanted to go over it first internally (even I didnt have time to review it so far) and than share it broadly for feedback and comments.
I will affect also networking and installer teams. We'll ask for more feedback early next week.